.ci/pipelines/e2e-tests-ocp-all-but-latest.Jenkinsfile

```
@@ -0,0 +1,90 @@
+// This library overrides the default checkout behavior to enable sleep+retries if there are errors
+// Added to help overcome some recurring github connection issues
+@Library('apm@current') _
+
+def failedTests = []
+def lib
+
+pipeline {
+
+    agent {
+        label 'linux'
+    }
+
+    options {
+        timeout(time: 48, unit: 'HOURS')
```

`cloud-on-k8s-e2e-tests-ocp` uses a 600 min timeout. I' keep this value and therefore multiply by 5.
```suggestion
        timeout(time: 50, unit: 'HOURS')
```